test_common/webmock.rb

```diff
@@ -0,0 +1,6 @@
+require 'webmock/cucumber'
```

Comment:
We dont use webmock ?  You dont need this

Reply:
updated